front/app/containers/EventsPage/CurrentAndUpcomingEvents.tsx

```diff
@@ -18,6 +18,7 @@ const CurrentAndUpcomingEvents = ({
     title={formatMessage(messages.upcomingAndOngoingEvents)}
     fallbackMessage={messages.noUpcomingOrOngoingEvents}
     eventsTime="currentAndFuture"
+    onClickTitleGoToProjectAndScrollToEvent={true}
   />
```

Comment:
This name is kind of long, but it's what is currently used, so I've just used the same.